packages/react-slider/src/components/RangedSlider/Spec.md

```diff
@@ -1,175 +0,0 @@
-# RangedSlider
```

Comment:
Should we preserve the previous spec somewhere, or would it be better to just look at history for reference if/when it's needed in the future?

Reply:
history works for me. the code is there, the old spec is there.